### PR TITLE
Updates documentation with Arquillian Forge Addon

### PR DESCRIPTION
**Pull request by lordofthejars**
#### Short description of what this resolves:

Updates documentation with Arquillian Forge Addon

#### Changes proposed in this pull request:

- Updates documentation with Arquillian Forge Addon
- New layout putting everything in `docs` folder
- Asciinema with Forge run
-

**Fixes**: #68

### CODE REVIEW

File: docs/skipDeployment.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+But if you set `skipDeployment` to *true*, lifecycle is slightly different:
+
+. Enrich `webapp` URL with Java system property or environment variable named `myservice.url` set in `@Environment`.
+. Executes contract tests against URL provided by `@Environment`.
+
+There is no additional "behind the scenes" Arquillian magic involved.
+Notice that with a simple attribute you can enable/disable how Arquillian behaves regarding the deployment lifecycle, and how you can reuse same test (DRY) for different scenarios.
+
+TIP: If you want to enable skipDeployment feature, you can set Java system property or environment variable `arq.extension.algeron-provider.SkipDeployment` to true or by using `${env.skipDeployment:false}` form.
```
Comment (bartoszmajsak) [inline docs/skipDeployment.adoc]: `arq.extension.algeron-provider.SkipDeployment` - should Skip be uppercase? Does it even matter?

File: docs/consumer.adoc [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+                .path("/")
+                .method("GET")
+                .willRespondWith()
+                .status(200)
+                .headers(header)
+                .body("{\"responsetest\": true, \"name\": \"harry\"}")
+                .toFragment(); // <4>
+    }
+
+    @EJB // <5>
```
Comment (bartoszmajsak) [inline docs/consumer.adoc]: Just a personal note - I wouldn't use `@EJB` - it's not considered as cool technology anymore :)

File: docs/provider.adoc [hunk 3]
```diff
@@ -0,0 +1,332 @@
+= Arquillian Algeron Pact Provider
```
Comment (bartoszmajsak) [inline docs/provider.adoc]: As discussed in the chat - maybe we don't use "Arquillian Algeron" prefix, as it's documentation is for "Arquillian Algeron" anyway :)

Comment (lordofthejars) [inline docs/provider.adoc]: I leave as this as we discussed later right?

**Pull request merged**